examples/local/cluster.tmpl.yaml

```diff
@@ -67,9 +67,9 @@ spec:
       etcd: "${AWS_API_HOSTED_ZONE}"
       ingress: "${AWS_INGRESS_HOSTED_ZONE}"
     vpc:
-      cidr: "10.0.0.0/16"
-      privateSubnetCidr: "10.0.0.0/19"
-      publicSubnetCidr: "10.0.128.0/20"
+      cidr: "10.1.5.0/24"
```

Comment:
Will this work if two of us (or CI in two separate branches) try this at the same time? Anyway :+1: for the mask change

Reply:
Yes collisions will still happen, but should be easy to solve by changing the third digit of the CIDR.